cpt.yaml

```diff
@@ -64,18 +64,10 @@ tools:
     owner: cpt
     tool_panel_section_label: Annotation
 
-  - name: cpt_intersect_adjacent
```

Comment:
We can not remove tools, they need to stay for reproducibility ... we could hide them if this is useful.

Please let them listed here and add a comment that they are deprecated.